opencti-platform/opencti-front/src/utils/Charts.js

```diff
@@ -328,25 +443,8 @@ export const horizontalBarsChartOptions = (
           navigate(`${link}/${entityId}`);
         }
       },
-      mouseMove: (event, chartContext, config) => {
```

Comment:
you shouldn't remove this part of code, it enables the cursor to be 'pointer' for parts of horizontal bars with redirection on click, and to be 'default' for parts with no redirection

Reply:
Thank you for your feedback :) It's fixed